src/main/java/storage/DataFileWriter.java

```diff
@@ -207,4 +215,28 @@ private void writeToFile(Path fileDirectory, Document xmlFileContents)
         transformer.transform(fileSource, fileResult);
     }
 
+    /**
+     * Gets the current settings and parses them into strings.
+     */
+    private void saveSettings() {
+        boolean isDisplayingHelpMessages = settings.getDisplayingHelpMessages();
+        int colorOption = settings.getColorOption();
+        StringBuilder saveData = new StringBuilder();
+        saveData.append("COLOR ").append(colorOption).append("\n");
+        saveData.append("HELPMESSAGE ").append(isDisplayingHelpMessages).append("\n");
```

Comment:
may wanna seperate each .append() to a new line for readability

Reply:
good point! will do